src/commands/dev/reload.ts

```diff
@@ -0,0 +1,85 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { codeBlock } from '@discordjs/builders';
+import { CommandInteraction } from 'discord.js';
+import { execSync } from 'child_process';
+import ApplicationCommandHandler from '@handlers/ApplicationCommandHandler';
+import ComponentHandler from '@handlers/ComponentHandler';
+
+export async function run(client: FluorineClient, interaction: CommandInteraction) {
+    await interaction.deferReply({ ephemeral: true, fetchReply: true });
```

Comment:
```suggestion
    await interaction.deferReply({ ephemeral: true });
```
we're not doing anything with the fetched reply